pages/common/sbcl.md

```diff
@@ -0,0 +1,12 @@
+# sbcl
+
+> High performance Common Lisp compiler.
+> More information: <http://www.sbcl.org/>.
+
+- Open an SBCL interactive shell (REPL):
+
+`sbcl`
+
+- Execute a Lisp script:
+
+`sbcl --script {{script.lisp}}`
```

Comment:
If a command takes a path, it's better to show that it can by adding some directories ;)
```suggestion
`sbcl --script {{path/to/script.lisp}}`
```